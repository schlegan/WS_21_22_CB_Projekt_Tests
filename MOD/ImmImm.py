print(420 % 69) # 6
print(0 % 69) # 0
print(-420 % 69) # -6, wegen dem Assemblyverhalten.
print(420 % -69) # 6, wegen dem Assemblyverhalten.
print(-420 % -69) # -6
print(0 % -69) # 0
