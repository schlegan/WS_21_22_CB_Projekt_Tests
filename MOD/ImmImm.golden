60-66-60
